### PR TITLE
[IMP] project: don't copy task's date on duplicate task or project

### PR DESCRIPTION
**Pull request by auon-odoo**
task-4295758

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/project/tests/test_project_base.py [hunk 1]
```diff
@@ -523,3 +523,19 @@ def test_archived_duplicate_task(self):
         task.active = False
         copy_task2 = task.copy()
         self.assertTrue(copy_task2.active, "Archived task should be active when duplicating an archived task")
+
+    def test_duplicate_doesnt_copy_date(self):
+        """ Test to check duplication of an archived task.
+            The duplicate of an archived task should be active.
+        """
```
Comment (deneuvillem) [inline addons/project/tests/test_project_base.py:530]: I think you've copy pasted this text by error :santa:

File: addons/project/tests/test_project_base.py [hunk 2]
```diff
@@ -523,3 +523,19 @@ def test_archived_duplicate_task(self):
...
+        project = self.env['project.project'].create({
```
Comment (deneuvillem) [inline addons/project/tests/test_project_base.py:531]: Can't we we set a `date_start` and a `date` to this project ?

File: addons/project/tests/test_project_base.py [hunk 3]
```diff
@@ -523,3 +523,19 @@ def test_archived_duplicate_task(self):
...
+            'name': 'Project',
+        })
+        task = self.env['project.task'].create({
```
Comment (deneuvillem) [inline addons/project/tests/test_project_base.py:534]: Same but for task, add a `date_deadline` ?

File: addons/project/tests/test_project_base.py [hunk 4]
```diff
@@ -523,3 +523,19 @@ def test_archived_duplicate_task(self):
...
+            'name': 'Task',
+            'project_id': project.id,
+        })
+        project_copy = project.copy()
+        self.assertFalse(project_copy.date, "The project's date fields shouldn't be copied on project duplication")
```
Comment (deneuvillem) [inline addons/project/tests/test_project_base.py:539]: We can make the check on `date_start` too :innocent:

**Review by deneuvillem — COMMENTED** [review]
Thanks for your work :jack_o_lantern: I left some comments about your test :)

**Review by xavierbol — COMMENTED** [review]
LGTM, thanks for your work. 🙂 

robodoo r+

**Comment by xavierbol** [comment]
robodoo delegate+

**Comment by xavierbol** [comment]
robodoo r+

**Pull request closed without merging**